bin/cli.js

```diff
@@ -22,10 +22,9 @@ program
             try {
                 await brs.execute(brsFiles, { root: program.root });
             } catch (err) {
-                if (err.messages && err.messages.length) {
-                    err.messages.forEach(message => console.error(message));
-                } else {
-                    console.error(err.message);
+                // rethrow unhandled errors
+                if (!err.location && !err.messages) {
+                    throw err;
```

Comment:
Any particular reason to re-throw errors here?  It's unclear to me what that provides over logging `err.message` like we do, since this file is only executed from a shell via `brs <args>`.

Reply:
Let me check again, maybe that's incorrect.

Reply:
The rethrow is only for non BRS errors, so that should only happen for a real runtime exception.

Reply:
I think Sean is saying that this is the `brs` cli, so either way we'll be logging the error to the command line and exiting with error (whether we re-throw, or just use `console.error`). What is the difference in intended output with these changes?